utils/mwa_metadb_utils.py

```diff
@@ -148,6 +142,31 @@ def getmeta(service='obs', params=None):
     return result
 
 
+def get_files(obsid, meta=None):
+    """
+    Queries the metadata to find all the file names
+    Parameters:
+    -----------
+    obsid: str
+        The ID (gps time) of the observation you are querying
+    meta: dict
+        The output of the getmeta function. This is an optional input that can
+        be used if you just want to extract the relevant info and save a 
+        metadata call
+    
+    Output:
+    -------
+    files: list
+        A list of all the file names
+    """
+    if meta is None:
+        files_meta = getmeta(servicetype='metadata', service='data_files', params={'obs_id':str(obsid)})
+    else:
+        files_meta = meta['files']
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing whitespace](https://app.codacy.com/app/CIRA-Pulsars-and-Transients-Group/vcstools/pullRequest?prid=4557977)